tests/workload/loader_test.py

```diff
@@ -1307,10 +1307,10 @@ def test_sets_absolute_path(self, path_exists):
         cfg = config.Config()
         cfg.add(config.Scope.application, "benchmarks", "local.dataset.cache", "/data")
 
-        default_test_procedure = workload.TestProcedure("default", default=True, schedule=[
+        default_test_procedure = workload.Procedure("default", default=True, schedule=[
             workload.Task(name="index", operation=workload.Operation("index", operation_type=workload.OperationType.Bulk), clients=4)
         ])
-        another_test_procedure = workload.TestProcedure("other", default=False)
+        another_test_procedure = workload.Procedure("other", default=False)
         t = workload.Workload(name="u", test_procedures=[another_test_procedure, default_test_procedure],
```

Comment:
You'll need to also change occurrences like `test_procedures`, `another_test_prcedure`, `default_test_procedure`. However, I think your approach is good if you plan to do a commit for each variation in this PR. For example, in the latest commit, you're renaming only `TestProcedure` -> `Procedure`. In subsequent commits, you can dedicate a commit to renaming instances of `test_procedure` or `test-procedure`.